admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,40 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. You can assign a pod or set of pods to specific nodes, or limit a project to only run on a desired set of nodes.
+
+For example, if you have a OpenShift topology with multiple regions, node selectors help restrict specific OpenShift projects to only deploy pods into a specific region. 
+
+To create a project with a node selector, use the `--node-selector` option. For example, the following creates a new project named `MyProject`, but runs only on the nodes labeled `user-node` and `east`:
+
+====
+----
+$ oadm new-project MyProject --node-selector='type=user-node,region=east'
+----
+====
+
+This adds an `annotation` section to the project configuration:
+
+====
+----
+...
+metadata:
+  annotations:
+    openshift.io/node-selector: type=user-node,region=east
+...
+----
+====
+
+Alternatively, edit the *_/etc/origin/master/master-config.yaml_* master configuration file and add a node selector value. This adds the node selector to all current projects without a `nodeSelector` value, as well as all future projects:
+
+====
+----
+...
+projectConfig:
+  nodeSelector: "type=user-node,region=east"
+...
+----
+====
```

Comment:
Should mention you would need to restart the **atomic-openshift-master** service after making this change for it to take effect.